cmd/bulk_load_influx/main.go

```diff
@@ -184,11 +184,16 @@ func (l *InfluxBulkLoad) CreateDb() {
 		log.Fatal(err)
 	}
 
-	if len(existingDatabases) > 0 {
+	if _, ok := existingDatabases["_internal"]; (ok == true && len(existingDatabases) > 1) || (ok == false && len(existingDatabases) > 0) {
```

Comment:
```suggestion
	if _, ok := existingDatabases["_internal"]; (ok && len(existingDatabases) > 1) || (!ok && len(existingDatabases) > 0) {
```

Reply:
Or, instead of the complicated bool check could we:
1. `delete(existingDatabases, "_internal")`
2. check `len(existingDatabases) > 0`

Reply:
Great suggestion, I like it